flink-yarn-tests/src/main/java/org/apache/flink/yarn/YARNSessionFIFOITCase.java

```diff
@@ -114,9 +108,9 @@ public void testClientStartup() {
 						"-n", "1",
 						"-jm", "768",
 						"-tm", "1024",
-						"-s", "2" // Test that 2 slots are started on the TaskManager.
+						"-s", "1" // Test that 1 slots are started on the TaskManager.
 				},
-				"Number of connected TaskManagers changed to 1. Slots available: 2", null, RunTypes.YARN_SESSION, 0);
+				"Number of connected TaskManagers changed to 1. Slots available: 1", null, RunTypes.YARN_SESSION, 0);
```

Comment:
Can you set the number of slots back to 2 and move the test to the other class as well?